### DISCUSSION TITLE
ADDING FERTILIZER TYPE IN THE "FERTILIZE ON FIXED DATE (AV)"

### DISCUSSION THREAD

**Discussion by Malitoz** [APSIMInitiative/ApsimX#8788] [General]
Hello,

I used CAN as a topdressing, but I cannot see it in the fertilizer type list. Is it possible to add it? Additionally, which other fertilizer is similar to this?

**Comment by hut104** [comment]
There are a range of commercial fertilisers referred to as Calcium Ammonium Nitrate (google it for example).
It may be difficult to have a single fertiliser for all users.
Perhaps add your CAN using two commands and add the amounts for NO3 and NH4 that you know are in whatever commercial product you have been using.
________________________________
From: Lucas Mutuma ***@***.***>
Sent: Wednesday, 27 March 2024 2:14 AM
To: APSIMInitiative/ApsimX ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [APSIMInitiative/ApsimX] ADDING FERTILIZER TYPE IN THE "FERTILIZE ON FIXED DATE (AV)" (Discussion #8788)


Hello,

I used CAN as a topdressing, but I cannot see it in the fertilizer type list. Is it possible to add it? Additionally, which other fertilizer is similar to this?

**Reply by Malitoz** [reply]
Thank you @hut104, very helpful.
